cpu/kinetis/dist/check-fcfield-bin.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+#
+# Flash configuration field check script for Freescale Kinetis MCUs.
+#
+# This script is supposed to be called from RIOTs
+# unified OpenOCD script (dist/tools/openocd/openocd.sh).
+#
+# Syntax: check-fcfield-hex.sh $HEXFILE
+#
+# @author       Jonas Remmert <j.remmert@phytec.de>
+# @author       Johann Fischer <j.fischer@phytec.de>
+# @author       Joakim Nohlgård <joakim.nohlgard@eistec.se>
+# @author       Francisco Molina <francisco.molina@inria.fr>
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 BINFILE"
+    echo "Checks the flash configuration field protection bits to avoid flashing a locked image to a Kinetis MCU (protection against accidental bricking)."
+    exit 2
+fi
+
+if [ -n "${IMAGE_OFFSET}" ]; then
```

Comment:
Its not really about jumping to invalid offsets or not. What we want to do is verify the content of the fcfield, if its over written or has different content than `fffffffffffffffffffffffffe..ffff` then the flash by get blocked and not be writable. 

I use `IMAGE_OFFSET` because when flashing at an offset the the fcfield is not located at the same position so I need to dissasemble different parts of the bin/hex file.

Reply:
No, you can brick it if the values in the fcfield are not the expecte ones. But since they are only read at reset the only ones that are read are in 0x400-0x40f. Si this shouldn't be needed indeed.

Reply:
From the `jumping`  I was refering to the `fcfield` at `jumpaddr + 0x400-0x40f`. Checking if they were considered and if the offseted one could brick it.

But now that I read about these bytes, "prevent reaing rom/prevent writing rom" and that they should be indeed at 0x00000400 there should not be any need to check them for the offset.

Proper handling `0 < IMAGE_OFFSET < 0x500` is not needed for the moment, just providing a "not handled for the moment" would be enough I think.